src/main/java/org/groomUniv/meet/common/security/CustomUserDetailsService.java

```diff
@@ -0,0 +1,35 @@
+package org.groomUniv.meet.common.security;
+
+import lombok.RequiredArgsConstructor;
+import org.groomUniv.meet.oauth.entity.Member;
+import org.groomUniv.meet.oauth.enums.Role;
+import org.groomUniv.meet.oauth.repository.MemberRepository;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CustomUserDetailsService implements UserDetailsService {
+    private final MemberRepository memberRepository;
+
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+        return memberRepository.findByEmail(username)
+                .map(this::createUserDetails)
+                .orElseThrow(() -> new UsernameNotFoundException("User not found"));
```

Comment:
```suggestion
                .orElseThrow(() -> new CoreException(GlobalErrorType.MEMBER_NOT_FOUND)
```

기존에 정의해놓은 exception 클래스를 활용할 수도 있을 것 같아요

Reply:
한번 활용해 보겠습니다!!